config/dataworks_schema.yml

```diff
@@ -128,6 +133,13 @@ required:
 additionalProperties: false
 components:
   schemas:
+    AccessContact:
+      type: object
+      properties:
+        name:
+          type: string
+        email:
+          type: string
```

Comment:
iirc there's a way to require at least one of these...is it maybe `oneOf`? seems like we'd want to at least ensure that you can't have an empty accesss contact

Reply:
@thatbudakguy updated